tests/integration/test_headers.py

```diff
@@ -0,0 +1,88 @@
+import pytest
```

Comment:
This looks much better now. Much easier to follow now we have extracted out into conftest.